tests/main/snap-confine-undesired-mode-group/task.yaml

```diff
@@ -0,0 +1,48 @@
+summary: the snap-{run,confine,exec} chain does not create files with undesired properties.
+prepare: |
+    # Install a snap with opengl and joystick plugs.
+    # This gives us all of the usual snap-confine configuration, along with all
+    # the cgroups that we create. 
+    snap pack test-snapd-app
+    snap install --dangerous ./test-snapd-app_1.0_all.snap
+    snap connect test-snapd-app:opengl
+    snap connect test-snapd-app:joystick
+execute: |
+    # Run the snap a non-root user.
```

Comment:
as a non-root user

Reply:
The review for the test is in https://github.com/snapcore/snapd/pull/7623